programmers/김서방-찾기.js

```diff
@@ -0,0 +1,4 @@
+function solution(seoul) {
+    var i = seoul.indexOf("Kim");
```

Comment:
```suggestion
    const idx = seoul.indexOf("Kim");
```
변수값이 변하지 않으면 const로 선언을하고, 변하는 값이면 let으로 선언을 합니다 
var는 es6이전에 사용하던 변수선언방법으로 이제는 잘 사용을 안하는데, 
그 이유는  변수를 중복선언할 수 있고, 예를들어 for문안에서 선언된 변수는 for문안에서만 써먹을 수 있어야하는데, var를 사용하면 for문 밖에서도 적용이 되기때문에, 혼란스러워집니다.

또한 변수명으로 a, i, x 등 의미없는 단어는 선택하지마시고 변수만으로 무슨 역할을 하는지 알 수 있게 작명해주세요. idx는 index의 약어입니다 ~
[참고](https://hoondev.tistory.com/101)